src/meta/src/hummock/manager/mod.rs

```diff
@@ -878,243 +876,309 @@ impl HummockManager {
+                        &compact_task,
+                        deterministic_mode,
+                    );
+                    current_version.apply_version_delta(&version_delta);
+                    trivial_tasks.push(compact_task);
+                    trivial_move_count += 1;
+                    if trivial_move_count >= MAX_TRIVIAL_MOVE_TASK_COUNT {
+                        break;
+                    }
+                } else {
+                    let mut table_to_vnode_partition = match self
+                        .group_to_table_vnode_partition
+                        .read()
+                        .get(&compaction_group_id)
+                    {
+                        Some(table_to_vnode_partition) => table_to_vnode_partition.clone(),
+                        None => BTreeMap::default(),
+                    };
+                    table_to_vnode_partition
+                        .retain(|table_id, _| compact_task.existing_table_ids.contains(table_id));
+
+                    if group_config.compaction_config.split_weight_by_vnode > 0 {
+                        for table_id in &compact_task.existing_table_ids {
+                            compact_task
+                                .table_vnode_partition
+                                .insert(*table_id, vnode_partition_count);
+                        }
+                    }
+                    compact_task.table_watermarks = current_version
+                        .safe_epoch_table_watermarks(&compact_task.existing_table_ids);
+
+                    if self.env.opts.enable_dropped_column_reclaim {
+                        compact_task.table_schemas = match self.metadata_manager() {
+                            MetadataManager::V1(mgr) => mgr
+                                .catalog_manager
+                                .get_versioned_table_schemas(&compact_task.existing_table_ids)
+                                .await
+                                .into_iter()
+                                .map(|(table_id, column_ids)| {
+                                    (table_id, TableSchema { column_ids })
+                                })
+                                .collect(),
+                            MetadataManager::V2(_) => {
+                                // TODO #13952: support V2
+                                BTreeMap::default()
+                            }
+                        };
+                    }
+
+                    compact_task_assignment.insert(
+                        compact_task.task_id,
+                        CompactTaskAssignment {
+                            compact_task: Some(compact_task.clone()),
+                            context_id: META_NODE_ID, // deprecated
+                        },
+                    );
+
+                    pick_tasks.push(compact_task);
+                    break;
                 }
+
+                stats = LocalSelectorStatistic::default();
```

Comment:
Will we miss `stats.report_to_metrics`?

Reply:
Because if we select a "trivial move task", we do not need report any  metrics for selector